From 98dd35ece2d68367506e19fc2a91aa3d678825bf Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:36:04 -0400
Subject: [alsa] seq/seq_memory: Fix closing brace followed by if

Message-id: <140972823573-349-patchset-jkysela@redhat.com>
Patchwork-id: 91045
O-Subject: [RHEL 7.1 PATCH 348/442] ALSA: seq: seq_memory.c: Fix closing brace followed by if
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Add a newline and, while at it, remove a space and redundant braces.

Signed-off-by: Rasmus Villemoes <linux@rasmusvillemoes.dk>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: b245a822a45915f63197d81cb899132e78f29fd8)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/core/seq/seq_memory.c b/sound/core/seq/seq_memory.c
index 1e206de..ba8e4a6 100644
--- a/sound/core/seq/seq_memory.c
+++ b/sound/core/seq/seq_memory.c
@@ -101,9 +101,9 @@ int snd_seq_dump_var_event(const struct snd_seq_event *event,
    len -= size;
   }
   return 0;
- } if (! (event->data.ext.len & SNDRV_SEQ_EXT_CHAINED)) {
-  return func(private_data, event->data.ext.ptr, len);
  }
+ if (!(event->data.ext.len & SNDRV_SEQ_EXT_CHAINED))
+  return func(private_data, event->data.ext.ptr, len);
 
  cell = (struct snd_seq_event_cell *)event->data.ext.ptr;
  for (; len > 0 && cell; cell = cell->next) {
-- 
1.7.1